src/pages/ShopPages/Categories/CategoriesPage.tsx

```
@@ -0,0 +1,53 @@
+        CategoriesProvider.getCategories().then((res) => {
+            setCategories(res.data);
+        });
+    }, []);
+    return (
+        <>
+            <div className="grid grid-cols-5 gap-5">
+                {categories.map((category) => (
+                    <Link
+                        className="w-full text-center"
+                        to={`${category.id}/products`}
+                    >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)